packages/worker/src/api/controllers/global/users.ts

```diff
@@ -1,8 +1,9 @@
 import {
-  checkInviteCode,
+  getInviteCode,
+  deleteInviteCode,
   getInviteCodes,
   updateInviteCode,
-} from "../../../utilities/redis"
+} from "@budibase/backend-core/src/redis/invite"
```

Comment:
We can't use import statements like this sadly, it generally baffles esbuild and can cause weird issues in production - for that reason we don't really allow import statements like this for monorepo libraries.

You can instead import it through the default import like `import { redis } from "@budibase/backend-core"` then use the commands like `redis.checkInviteCode`